tools/tuf/README.md

```diff
@@ -294,6 +294,32 @@ TIMESTAMP_PASSPHRASE_1PASSWORD_PATH="Private/UPDATES TIMESTAMP/password" \
 ACTION=release-to-production ./tools/tuf/releaser.sh
 ```
 
+### Doing a patch release of fleetd
+
+Patch releases follow the same process as releasing a minor version, except instead of checking out the `main` branch of fleet locally, you check out a patch branch of fleet, e.g.:
+
+```
+git checkout rc-minor-fleet-v.4.65.1
```

Comment:
One more thing: Let's use a different sample cut name: `rc-minor-fleetd-v1.41.0` (it will probably be done around the same day but AFAICS they should be separate from the fleet cut)